tests/CollectionTest.php

```diff
@@ -2062,6 +2066,43 @@ public function testIsVersioningEnabled()
         ));
         $this->assertFalse($this->database->col3->isVersioningEnabled());
     }
+
+    /**
+     * @throws Exception
+     * @throws \MongoException
+     */
+    public function testRenameNonExistentCollection()
+    {
+        $this->expectException(Exception::class);
+        $this->expectExceptionCode(26);
+        $this->expectExceptionMessage('Error: #26: source namespace does not exist');
+
+        // set test collection
+        $this->collection = $this->database->getCollection('test');
```

Comment:
why you overwrite collection from setUp. If you don't use, just crete local var.